lightning/src/events/mod.rs

```diff
@@ -250,12 +258,14 @@ impl_writeable_tlv_based_enum_upgradable!(ClosureReason,
 	(1, FundingTimedOut) => {},
 	(2, HolderForceClosed) => {},
 	(6, CommitmentTxConfirmed) => {},
-	(4, CooperativeClosure) => {},
+	(4, LegacyCooperativeClosure) => {},
```

Comment:
Unrelated nit:

Since we are touching these lines, should we think about reordering these enums? I think it's pretty odd to write 6th before 4th.

```suggestion
        (4, LegacyCooperativeClosure) => {},
        (6, CommitmentTxConfirmed) => {},
	
```